### DISCUSSION TITLE
MCU 'eboard' shutdown: Timer too close

### DISCUSSION THREAD

**Discussion by DokerRS** [xblax/flashforge_ad5m_klipper_mod#265] [General]
Recently, I started catching the error "MCU 'eboard' shutdown: Timer too close" very often. I use FF 2.7.6 firmware and klippermod v00.05-beta in conjunction with klipperscreen and camera. 74% of the RAM is used in the idle state. This error occurs only on the mod at different gcode and different printing stages. I have already tried to turn off the camera and klipperscren, but this did not solve the problem, while if you switch to stock firmware, everything works well and does not freeze. I ask for help with this issue, because I am not the first and only one who has encountered this problem.
[moonraker.log](https://github.com/user-attachments/files/17825888/moonraker.log)
[klipper.log](https://github.com/user-attachments/files/17825891/klipper.log)

**Comment by AlexStrew** [comment]
Same problem

**Comment by DokerRS** [comment]
I reset the printer to firmware 2.6.6, installed klipper with guppyscreen. I pressed the parking button of the print head and table and immediately caught the error MCU 'mcu' shutdown: Timer too close. How can this be fixed?
[klipper(2).log](https://github.com/user-attachments/files/17832791/klipper.2.log)
[moonraker(2).log](https://github.com/user-attachments/files/17832794/moonraker.2.log)

**Comment by Berkano24** [comment]
I also see this error from time to time, often during printing.

**Comment by KaruroChori** [comment]
Closing to avoid duplication. The issue was also reported in https://github.com/xblax/flashforge_ad5m_klipper_mod/discussions/245
